main.go

```diff
@@ -15,6 +18,88 @@ import (
 var telemetryDB = make(map[string]TelemetryData)
 var listIPs = make(map[int]string)
 
+// Request is either:
+//   - Payload Ops (Ground)
+//   - Uplink/Downlink (Ground)
+func handleScenarioOne(c *gin.Context, r RedirectRequest) {
+	// Create a new HTTP request with the verb, URI and
+	// data as specified by the RedirectRequest
+	req, err := http.NewRequest(r.Verb, r.URI, bytes.NewReader([]byte(r.Data)))
+	if err != nil {
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	// Send this request using the default HTTP Client
+	// and receive a response
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	// Send the response back to the user
+	io.Copy(c.Writer, resp.Body)
+}
+
+// Request is either:
+//   - Payload Ops (Space)
+//   - CNDH (Space)
+//   - Uplink/Downlink (Space)
+//   - Payload Ops (Center)
+func handleScenarioTwo(c *gin.Context, r RedirectRequest) {
```

Comment:
This scenario is when we can't DIRECTLY send a request to a module.

The function needs to:

1. Check the destination
2. If the destination is modules 1-3, pass the data along to the Uplink/Downlink's /send/ route (The request will look identical to the one we got from /receive/, except it will be to Uplink/Downlink's /send/ route).
3. If the destination is module 7, use whatever route ground payload ops has created to route requests (You may need to ping Artem/Ali and check if they have a route for us to send this to module 7).